test/unit/asset-fs-array-expr-node-require/output.js

```diff
@@ -1,5 +0,0 @@
-[
-  "package.json",
-  "test/unit/asset-fs-array-expr-node-require/asset1.txt",
-  "test/unit/asset-fs-array-expr-node-require/input.js"
-]
```

Comment:
I was thinking the output.js file was necessary for each test fixture 🤔

Reply:
It's still necessary just was renamed https://github.com/vercel/nft/blob/main/test/unit/asset-fs-array-expr-node-prefix/output.js guess it shows as a delete because the content changed more than git expects.

Reply:
Oh I see, git didn't track it as a rename since the contents changed too much 👍